code/modules/mob/living/carbon/life.dm

```diff
@@ -146,6 +146,15 @@
 
 	//CRIT
 	if(!breath || (breath.total_moles() == 0) || !lungs)
+		if(lungs && HAS_TRAIT(lungs, TRAIT_SPACEBREATHING))
```

Comment:
Wouldn't it be nicer if this was evaluated automatically based on the lungs, rather than an extra flag? the information is available from the lungs already anyway

Reply:
huh